File: app/src/main/java/in/woloo/www/giftcard/model/RequestPointsResponse.java
package in.woloo.www.giftcard.model;

import com.google.gson.annotations.SerializedName;

public class RequestPointsResponse{

	@SerializedName("data")
	private Data data;

	public Data getData(){
		return data;
	}

	public class Data{

		@SerializedName("message")
		private String message;

		@SerializedName("order_id")
		private String orderId;

		@SerializedName("status")
		private String status;

		public String getMessage(){
			return message;
		}

		public String getOrderId(){
			return orderId;
		}

		public String getStatus(){
			return status;
		}
	}
}

File: app/src/main/java/in/woloo/www/more/giftcard/model/RequestPointsResponse.kt
package `in`.woloo.www.more.giftcard.model

import com.google.gson.annotations.SerializedName

class RequestPointsResponse {
    @SerializedName("data")
    val data: Data? = null

    inner class Data {
        @SerializedName("message")
        val message: String? = null

        @SerializedName("order_id")
        val orderId: String? = null

        @SerializedName("status")
        val status: String? = null
    }
}